package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "messagebird",
-  "version": "2.2.0",
+  "version": "3.1.0",
```

Comment:
I don't think this is a breaking change right? Following semver I think this should be a `patch` instead? since also no new features are added